server/static/js/uni/ajax.js

```diff
@@ -18,14 +18,18 @@ const RequestMethod = {
  * @param method                - Request method. Can be "GET" or "POST"
  * @param callback              - Callback function
  * @param failure_processor     - Function that will process the errors, should they occur
+ * @param beforeSend            - Function used to display the preloader during the execution of the request
```

Comment:
Well, in scope of this particular task it might be used to display loading spinner, but it's not limited just to that.
Correct the description please - it just calls given function before executing the AJAX request.